scripts/asm-installer/cloudbuild-master.yaml

```diff
@@ -51,6 +51,24 @@ steps:
   - '_CI_ASM_PKG_LOCATION=${_PKG_LOCATION}'
   - '_CI_ASM_KPT_BRANCH=${_ASM_PKG_BRANCH}'
 
+- name: 'gcr.io/$PROJECT_ID/${_IMAGE_NAME}'
+  dir: 'scripts/asm-installer'
+  id: 'setup-long-term-cluster-for-environ-test'
+  entrypoint: '/bin/bash'
+  args:
+  - '-c'
+  - >
+    ./tests/setup_longterm_cluster
+  waitFor:
+  - 'fetch-secrets'
```

Comment:
This file doesn't use waitFor, all of the tests are sequential.

Reply:
What is the reason for no waitFor if they are setting up/running different clusters?

Reply:
The reason why we changed to running on one cluster in the first place is because cluster creation/IAM operations would interfere with each other or time out.

Reply:
I don't think with two clusters running tests separately, there should be operations conflicting each other. But for now, I remove that as it is not in scope of what this PR is trying to do.